src/index.js

```diff
@@ -0,0 +1,122 @@
+if(!global._babelPolyfill) { require('babel-polyfill'); }
+
+import Proto from 'uberproto';
+import filter from 'feathers-query-filters';
+import errors from 'feathers-errors';
+import { nfcall, getSelect, multiOptions, mapItems } from './utils';
+
+// Create the service.
+class Service {
+	constructor(options = {}) {
+		if(!options.db) {
```

Comment:
To normalize with the other db adapters this should be `Model`. Then it will make it much easier for us to generate.